internal/dispatch/dispatch.go

```diff
@@ -86,23 +82,23 @@ type LookupSubjects interface {
 	) error
 }
 
-// HasMetadata is an interface for requests containing resolver metadata.
-type HasMetadata interface {
+// DispatchableRequest is an interface for requests.
+type DispatchableRequest interface {
```

Comment:
the interface does not say anything about dispatch tho, more like `MetadataProvider`

Reply:
Sure, but it is only used for dispatchable requests